srcpkgs/reaver/template

```diff
@@ -1,29 +1,20 @@
 # Template file for 'reaver'
 pkgname=reaver
-version=1.6.5
-revision=2
-build_style=gnu-configure
+version=1.6.6
+revision=1
 build_wrksrc="src"
-#conf_files="/etc/reaver/reaver.db"
+build_style=gnu-configure
 makedepends="libpcap-devel"
 short_desc="Brute force attack tool against Wifi Protected Setup (WPS)"
 maintainer="cipr3s <cipr3s@gmx.com>"
-license="GPL-2"
+license="GPL-2.0-or-later"
 homepage="https://github.com/t6x/reaver-wps-fork-t6x"
 distfiles="https://github.com/t6x/reaver-wps-fork-t6x/releases/download/v${version}/${pkgname}-${version}.tar.xz"
-checksum=342e9d265cf459bd2387205b73a63d1fc7582e268f0e9aec20613f3ec11b6a6b
+checksum=e329a0da0b6dd888916046535ff86a6aa144644561937954e560bb1810ab6702
 
 post_extract() {
 	case "$XBPS_TARGET_MACHINE" in
 	*-musl) find -type f -exec sed -i "{}" -e"s;u_char;unsigned char;g" \;
 		;;
 	esac
 }
-
-#do_install() {
-#	# The Makefile's install target is broken (ignores destdir)
-#	vmkdir etc/${pkgname}
-#	vinstall ${pkgname}.db 644 etc/${pkgname}
```

Comment:
FWIW the current version of the package doesn't have the db file. Should that be fixed?

Reply:
Maybe I'm confused here but deleting a commented-out section should be a no-op.

Reply:
Just pointing out that maybe commenting the whole thing out wasn't the correct course of action. `do_install` installed a db file, that doesn't seem to be in the package anymore.

Reply:
Ah, fair. Updated the commit message to document that I'm cleaning up old badness.